docs/patterns/components/RadioCardGroup/index.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Demo from '../../../Demo';
+import {
+  RadioCardGroup,
+  Code,
+  FieldWrap
+} from '../../../../packages/react/lib';
+
+const labelDescription =
+  'A label for the radio card group is required. This means you must provide either an aria-label or aria-labelledby prop.';
+
+const DemoRadioCardGroup = () => {
+  return (
+    <>
+      <h1>RadioCardGroup</h1>
+      <p>
+        <strong>NOTE:</strong> All form fields should be rendered as children of
+        a{' '}
+        <Link to="/components/FieldWrap">
+          <code>{`<FieldWrap />`}</code> component
+        </Link>
+        . This means ALL fields in a form should be wrapped by 1 or more (in the
+        case of "groups" of fields) FieldWrap components.
+      </p>
+
+      <h2>Demo</h2>
+
+      <div>
+        <h3 id="coffee-label">Do you like coffee?</h3>
+        <RadioCardGroup
+          name="coffee"
+          defaultValue="yes"
+          aria-labelledby="coffee-label"
+          radios={[
+            {
+              id: 'yes',
+              value: 'yes',
+              label: 'Yes',
+              cardImg: (
+                <div
+                  style={{
+                    backgroundColor: 'green',
+                    height: 100,
+                    width: 100,
+                    borderRadius: 50
+                  }}
+                ></div>
+              ),
+              cardIcon: 'check-circle'
+            },
+            {
+              id: 'no',
+              value: 'no',
+              label: 'No',
+              disabled: true,
+              cardImg: (
+                <svg viewBox="0 0 100 100">
```

Comment:
Should maybe aria-hide this so screen readers won't announce a generic nameless image

```suggestion
                <svg viewBox="0 0 100 100" aria-hidden="true">
```